FoodBookApp/Handlers/NotificationHandler.swift

```diff
@@ -15,6 +15,7 @@ class NotificationHandler {
         { success, error in
             if success {
                 print("Access granted!")
+                self.sendLastReviewNotification(date: Date())
```

Comment:
question, why does the method need to be called here? wouldn't it send a notification right away since it's with the default date?

Reply:
It's called here bc once notifications are approved, it will schedule one for the first time since otherwise it would only send one if they make a review, so if they never do, it will never encourage them to. Also, this function receives today's date and within it adds up the four days, check the triggerDar variable:
```swift
func sendLastReviewNotification(date: Date) {
        
        let notificationIdentifier = "lastReviewNotification"

        cancelNotification(identifier: notificationIdentifier)
                    
        let triggerDate = Calendar.current.date(byAdding: .day, value: 4, to: date)!
        
        let trigger = UNCalendarNotificationTrigger(dateMatching: Calendar.current.dateComponents([.year, .month, .day, .hour, .minute, .second], from: triggerDate), repeats: false)
                
        let content = UNMutableNotificationContent()
        content.title = "We miss you..."
        content.body = "You haven't left a review in a while"
        content.sound = UNNotificationSound.default
        
        let request = UNNotificationRequest(identifier: notificationIdentifier, content: content, trigger: trigger)
        
        UNUserNotificationCenter.current().add(request)
    }

```

Reply:
I only changed where it's called (bc before it was triggered regardless of whether or not permission was given), not the way it works, it has always received the current date.

Reply:
I could move to outside the if, just in case they turn on notifications during that four day span. If they remain off, the notification won't be visible so it's not a problem. But either way the date is okay :)

Reply:
Okay, i understand!! Thank you!!

Reply:
@Laurarestrepo03 please re-request a review from me so i can approve!!

Reply:
Thank you! I saw you approved the PR, so I'll make that small change and merge :p